dev/com.ibm.ws.jsp/src/com/ibm/wsspi/jsp/tools/JspToolsOptionKey.java

```diff
@@ -1,14 +1,11 @@
 /*******************************************************************************
- * Copyright (c) 1997, 2004 IBM Corporation and others.
+ * Copyright (c) 1997, 2021 IBM Corporation and others.
```

Comment:
I'm assuming 2021 is when this code was first pushed up by Evan?

Reply:
Yes